src/elements/content-sidebar/__tests__/SidebarNavSign.test.tsx

```diff
@@ -1,80 +1,75 @@
 import * as React from 'react';
-import { render, fireEvent } from '@testing-library/react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
 import SidebarNavSign from '../SidebarNavSign';
 // @ts-ignore Module is written in Flow
 import FeatureProvider from '../../common/feature-checking/FeatureProvider';
+// @ts-ignore Module is written in Flow
+import messages from '../messages';
+// @ts-ignore Module is written in Flow
+import localize from '../../../../test/support/i18n.js';
+
+jest.unmock('react-intl');
 
 describe('elements/content-sidebar/SidebarNavSign', () => {
     const onClickRequestSignature = jest.fn();
     const onClickSignMyself = jest.fn();
+    const signLabel = localize(messages.boxSignRequestSignature.id);
+    const requestSignatureButtonText = localize(messages.boxSignRequestSignature.id);
+    const signMyselfButtonText = localize(messages.boxSignSignMyself.id);
 
     const renderComponent = (props = {}, features = {}) =>
         render(
             <FeatureProvider features={features}>
                 <SidebarNavSign {...props} />
             </FeatureProvider>,
+            {
+                wrapper: ({ children }: { children?: React.ReactNode }) => (
+                    <IntlProvider locale="en-US">{children}</IntlProvider>
+                ),
+            },
         );
 
-    test.each([true, false])('should render sign button', isRemoveInterstitialEnabled => {
-        const features = {
-            boxSign: {
-                isSignRemoveInterstitialEnabled: isRemoveInterstitialEnabled,
-            },
-        };
+    test('should render sign button', () => {
+        renderComponent();
 
-        const wrapper = renderComponent({}, features);
-        expect(wrapper.getByTestId('sign-button')).toBeVisible();
+        expect(screen.getByLabelText(signLabel)).toBeVisible();
     });
 
-    test('should call correct handler when sign button is clicked', () => {
-        const features = {
-            boxSign: {
-                isSignRemoveInterstitialEnabled: false,
-                onClick: onClickRequestSignature,
-            },
-        };
-        const { getByTestId } = renderComponent({}, features);
-
-        fireEvent.click(getByTestId('sign-button'));
+    test('should open dropdown with 2 menu items when sign button is clicked', () => {
+        renderComponent();
 
-        expect(onClickRequestSignature).toBeCalled();
-    });
+        fireEvent.click(screen.getByLabelText(signLabel));
```

Comment:
we should use userEvent instead of fireEvent, this goes for the below fireEvents too, you dont need to install the package for userEvent as its already provided, you just need to import it